fixham/internal/dict/dict.go

```diff
@@ -0,0 +1,46 @@
+package dict
```

Comment:
Don't need a whole package for this - can just plop it in `hashrelease` probably, since that's the only place it's used.